### ISSUE TITLE
Can't add music or mods

### ISSUE THREAD

**Issue by Midaychi** [diddlesnaps/starruler2#2]
Not sure if it's possible, but it would be neat to have some sort of plug or whatever so you can connect the music folder in data or mods to a folder in your home directory. 
As is right now, you can't modify snaps without re-doing the snap yourself, so you can only use the base game w/o mods and can't add in custom music or even the music if you own a copy of the game elsewhere.

**Comment by Midaychi** [comment]
maybe through this?
https://forum.snapcraft.io/t/layouts-re-mapping-snap-directories/1471

**Comment by lucyllewy** [comment]
Yeah layouts might help, though I think a symbolic link may work better and doesn't require support from snapd. (at least for the music scenario)

**Comment by LargePrime** [comment]
cannot currently mod portraits

**Comment by woeisme** [comment]
Hey @diddledani thanks for your great work first of all!
Could you consider adding this CC music to your snap 
https://opengameart.org/content/star-ruler-2-replacement-music ?
